package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@astrojs/starlight": "^0.32.2",
+    "@bomb.sh/args": "^0.3.1",
```

Comment:
Hmm do we need to install this?

Reply:
I have to check, but I'm pretty sure that without it twoslash won't be able to do its magic.

But maybe it can be moved into dev dependencies ? :thinking:

Reply:
I confirm, without it twoslash don't work.

But it can be moved to `devDependencies`

Reply:
yeah, it is required when working with twoslash.